packages/tables/backend/sirius-components-collaborative-tables/src/main/resources/schema/table.graphqls

```diff
@@ -54,6 +54,7 @@ type Column {
   isResizable: Boolean!
   hidden: Boolean!
   filterVariant: String!
+  index: Int
```

Comment:
Why such an index and why would it be optional? The order of the column is already given by the order in `Table#columns`, why would we need a second piece of information for that?